### ISSUE TITLE
Update to @angular/material (version 2 beta 5)

### ISSUE THREAD

**Issue by mickjol** [infra-geo-ouverte/igo2-lib#55]
We try update the are version of angular material.

But we receive the following error when we does.

igo.js:2672
Uncaught TypeError: material_1.MaterialModule.forRoot is not a function

**Comment by cbourget** [comment]
forRoot was deprecated in this version of Material. You need to upgrade IGO too to fix this issue. Note that the latest release of IGO brought a few changes that may raise errors in your application but can be fixed easily. If you need assistance to upgrade don't hesiate to ask. me.

**Comment by mbarbeau** [comment]
I created a release (0.4.2) that fixes this problem.

**Comment by mickjol** [comment]
I made the update. But I have some problem with me layers inside my DetailedContext.

**Comment by cbourget** [comment]
You have to move 'type' inside 'source'. Since this new release layers have been split into 'Layer' and 'DataSource' to better support non-spatial data.

**Comment by mickjol** [comment]
Nice it work,

**Comment by cbourget** [comment]
He are the commits involved:

https://github.com/infra-geo-ouverte/igo2-lib/commit/743e2d9ad92a519651b39c9bf264880931fad2e2
https://github.com/infra-geo-ouverte/igo2-lib/commit/d6fa71a15efaf05830b1a58d44b46337c280030d
https://github.com/infra-geo-ouverte/igo2-lib/commit/5013e1a42028378015babd9b7246ee09f6462c6f

And the complete list of commits since 0.3.3 (your version):

https://github.com/infra-geo-ouverte/igo2-lib/compare/0.3.3...0.4.2

**Comment by cbourget** [comment]
You need to do something like this:

```
const interventionDataSource = new FeatureDataSource({
  title: 'Interventions'
});

const interventionLayer = new VectorLayer(interventionDataSource, {
  zIndex: 20
});
this.map.addLayer(interventionLayer, false);
this.interventionSource = interventionDataSource.ol;
```

**Comment by mickjol** [comment]
Good, with this modification and the fix on 0.4.3 everything work.
